hack/common.sh

```diff
@@ -36,6 +36,10 @@ function os::build::host_platform_friendly() {
     echo "linux-64bit"
   elif [[ $platform == "linux/ppc64le" ]]; then
     echo "linux-powerpc64"
+  elif [[ $platform == "linux/arm64" ]]; then
+    echo "linux-aarch64"
```

Comment:
Understood.  The 64 bit arch arch comes in too many variations.
I'm fine with whatever he thinks.  I think the other alternative would be linux-arm64